x/token/msgs.go

```diff
@@ -0,0 +1,490 @@
+package token
+
+import (
+	"regexp"
+	"unicode/utf8"
+
+	"github.com/line/lbm-sdk/codec/legacy"
+	sdk "github.com/line/lbm-sdk/types"
+	sdkerrors "github.com/line/lbm-sdk/types/errors"
+)
+
+const (
+	TypeMsgTransfer     = "transfer"
+	TypeMsgTransferFrom = "transfer-from"
+	TypeMsgApprove      = "approve"
+	TypeMsgIssue        = "issue"
+	TypeMsgGrant        = "grant"
+	TypeMsgRevoke       = "revoke"
+	TypeMsgMint         = "mint"
+	TypeMsgBurn         = "burn"
+	TypeMsgBurnFrom     = "burn-from"
+	TypeMsgModify       = "modify"
+
+	ActionMint   = "mint"
```

Comment:
Hard coded type strings are no more in need.
```suggestion
	ActionMint   = "mint"
```